app/Domain/Trips/Console/DeleteExpiredTripsAndRelatedContentCommand.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Trips\Console;
+
+use App\Domain\Trips\Jobs\DeleteExpiredTripAndContentJob;
+use App\Domain\Trips\Models\Trip;
+use Illuminate\Console\Command;
+
+class DeleteExpiredTripsAndRelatedContentCommand extends Command
+{
+    protected $signature = 'trips:delete-expired-trips {dayToGoBack}';
+
+    protected $description = 'This command will delete any trips that are soft deleted and have been deleted for more than the days requested';
+
+    public function handle(): void
+    {
```

Comment:
I think you can define the type of an argument now in the signature using `{dayToGoBack:int}` so that you don't have to cast it here.

Reply:
just tried this it doesnt work sadly